README.md

```diff
@@ -108,6 +108,18 @@ It may be the case that the benchmarks require more resources than are configure
 
 ***Note***: Benchmarking RonDB with a docker-compose setup on a single machine may not bring optimal performance results. This is because both the mysqlds and the ndbmtds (multi-threaded data nodes) scale in performance with more CPUs. In a production setting, each of these programs would be deployed on their own VM, whereby mysqlds and ndbmtds will scale linearly with up to 32 cores. The possibility of benchmarking was added here to give the user an introduction of benchmarking RonDB without needing to spin up a cluster with VMs.
 
+## Permission Problems when mounting local files
+
+By default, we mount all benchmarking directories from the host filesystem, so that the results can be easily viewed locally. However, when not running on a Mac and running with uid:gid 1000:1000, one may run into permission problems when the benchmarking script attempts to write into these directories. We use UID/GID 1000:1000 (mysql:mysql) in the image because it mimics the cloud environment that we usually operate in at Hopsworks. Also, if we used root:root, the container could create files that a non-root host user may not be able to delete anymore. We considered using dynamic arguments as UID/GID, but since we use this Dockerfile in our CI to upload images to Dockerhub, and we did not want arbitrary UID/GIDs in our public images, we did not go along this route.
```

Comment:
I'm not sure I understand - can you post an alternative sentence?

Reply:
Perhaps change

`Also, if we used root:root, the container could create files that a non-root host user may not be able to delete anymore.`

to

`Also, no matter what (static) UID the container uses, it might create files that a non-root host user having another UID may not be able to delete anymore.`

But either way, if I'm successful at the `--gid` workaround, perhaps you'll want to scrap the entire heading anyway.

Reply:
Okay sure